controller/src/main/java/com/epam/esm/util/builder/CreateHypermediaLinkBuilder.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.esm.util.builder;
+
+import com.epam.esm.contoller.GiftCertificateController;
+import com.epam.esm.contoller.OrderController;
+import com.epam.esm.contoller.TagController;
+import com.epam.esm.contoller.UserController;
+import com.epam.esm.dto.CreateActionHypermedia;
+import org.springframework.hateoas.server.mvc.WebMvcLinkBuilder;
+
+public class CreateHypermediaLinkBuilder extends BuilderContainer<CreateActionHypermedia> {
+
+    public CreateHypermediaLinkBuilder(CreateActionHypermedia hypermedia) {
+        super(hypermedia);
+    }
+
+    public CreateHypermediaLinkBuilder buildNewCertificateLink(long id) {
+        hypermedia.add(WebMvcLinkBuilder
+                .linkTo(WebMvcLinkBuilder.methodOn(GiftCertificateController.class)
+                        .findSpecificCertificate(id))
+                .withRel("certificate"));
```

Comment:
It's good practice to move "magic" strings, numbers, etc to constants. Moreover you have several usages of "certificates"